src/SystemCommands-SourceCodeCommands/SycExtractTempCommand.class.st

```diff
@@ -57,5 +64,7 @@ SycExtractTempCommand >> prepareFullExecutionInContext: aSourceCodeContext [
 		request: 'New temp name'
 		initialAnswer: ''
 		title: 'Extract code to temp variable'.
-	tempName isEmptyOrNil ifTrue: [ CmdCommandAborted signal ]
+	tempName isEmptyOrNil ifTrue: [ CmdCommandAborted signal ].
```

Comment:
Now that we have driver, this logic is moved to the driver, so this code can be removed. Pay attention to update driver constructor so it is created without the new tempName.